myapp/bill/models.py

```diff
@@ -7,8 +7,8 @@ class Bills(models.Model):
     date = models.DateTimeField('Дата')
 
     def __str__(self):
-        return self.title
+        return f' {self.title} {self.price}р. {self.date}'
 
     class Meta:
```

Comment:
I think its not relevant name for this model

Reply:
yes, i`m agreed
and changed name